### ISSUE TITLE
Screen Reader: Many Pages: The link of Licenses is not focusable with Tab key, no role announced

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#74872]
## Action Performed
1. Using JAWS + Chrome, open the web page https://new.expensify.com/
2. Press Tab key and try to focus the 'licenses' link, located in the footer under the 'Get Started' header
3. Press the arrow keys to focus the control and note how it is announced.

Other Occurrences:
1. On Settings - About: the links Terms of Service and Privacy
2. On Workspace - Reports: the link 'extensive formulas'
3. On Workspace - Rules: the links 'categories' and 'tags'
4. On Workspace - Company Cards - Add cards: the link 'options'
5. On Create Report - Change payment currency: the link 'pricing page'
6. On Create Expense: the links 'download the app', email and 'add your number'

## Expected Result
The control should be focused with Tab key and announced with a link role.

## Actual Result
The link of Licenses is not focusable with Tab key, no link role announced.

HTML:
<span class="css-1jxf684 r-1loqt21" style="color: rgb(90, 176, 255); font-size: 9px; text-align: left; font-family: &quot;Expensify Neue&quot;, &quot;Windows Segoe UI Emoji&quot;, &quot;Noto Color Emoji&quot;, &quot;Custom Emoji Font&quot;; font-style: normal; font-weight: 400; line-height: 11px; text-decoration-color: rgb(90, 176, 255);">licenses</span>

## Area issue was found
Landing Page

## Failed Checkpoint
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User Impact
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested Resolution
Implement the control as a native <a> element with a valid href attribute.
https://www.w3.org/WAI/ARIA/apg/patterns/link/

## Environment(s)
Windows Windows 10 - Chrome

## Screenshots/Videos
<img width="1919" height="997" alt="Image" src="https://github.com/user-attachments/assets/0a9df9f7-dc3d-40c9-971b-2a65a9bb0a1c" />

https://github.com/user-attachments/assets/0150fe1d-582e-4145-9d58-481738db75bd

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6985829
**Issue reported by:** NA
**Slack conversation:** NA

**Comment by mallenexpensify** [comment]
`Accessibility` issue, will be addressed soon.

**Comment by mallenexpensify** [comment]
Accessibility, unassigning, will be addressed in bulk once all issues are created.
